pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/BaseClusterIntegrationTestSet.java

```diff
@@ -714,58 +714,29 @@ public void testReload(boolean includeOfflineTable)
     assertEquals(queryResponse.get("resultTable").get("dataSchema").get("columnNames").size(), schema.size());
     long numTotalDocs = queryResponse.get("totalDocs").asLong();
 
-    schema.addField(constructNewDimension(FieldSpec.DataType.INT, true));
-    schema.addField(constructNewDimension(FieldSpec.DataType.LONG, true));
-    schema.addField(constructNewDimension(FieldSpec.DataType.FLOAT, true));
-    schema.addField(constructNewDimension(FieldSpec.DataType.DOUBLE, true));
-    schema.addField(constructNewDimension(FieldSpec.DataType.STRING, true));
-    schema.addField(constructNewDimension(FieldSpec.DataType.INT, false));
-    schema.addField(constructNewDimension(FieldSpec.DataType.LONG, false));
-    schema.addField(constructNewDimension(FieldSpec.DataType.FLOAT, false));
-    schema.addField(constructNewDimension(FieldSpec.DataType.DOUBLE, false));
-    schema.addField(constructNewDimension(FieldSpec.DataType.STRING, false));
-    schema.addField(constructNewMetric(FieldSpec.DataType.INT));
-    schema.addField(constructNewMetric(FieldSpec.DataType.LONG));
-    schema.addField(constructNewMetric(FieldSpec.DataType.FLOAT));
-    schema.addField(constructNewMetric(FieldSpec.DataType.DOUBLE));
-    schema.addField(constructNewMetric(FieldSpec.DataType.BYTES));
-
-    // Upload the schema with extra columns
-    addSchema(schema);
+    addNewSchemaFields(schema);
     String tableNameWithTypeOffline = TableNameBuilder.forType(TableType.OFFLINE).tableNameWithType(rawTableName);
     String tableNameWithTypeRealtime = TableNameBuilder.forType(TableType.REALTIME).tableNameWithType(rawTableName);
-    // Reload the table
+    String reloadJob;
+    //tests that reload is needed on the table from controller api segments/{tableNameWithType}/needReload
```

Comment:
(minor) Add a space and capitalize, same for other places
```suggestion
    // Tests that reload is needed on the table from controller api segments/{tableNameWithType}/needReload
```